RecoPixelVertexing/PixelVertexFinding/plugins/WorkSpaceUtilities.h

```diff
@@ -0,0 +1,35 @@
+#ifndef RecoPixelVertexing_PixelVertexFinding_WorkSpace_h
+#define RecoPixelVertexing_PixelVertexFinding_WorkSpace_h
```

Comment:
We kept it as utilities since, in the whole majority of the files that define the SoALayout, several other useful methods are also being defined in those. Is this not recommended? Should they be separated in different Utilities and SoAHeterogeneousLayout files?

Reply:
I don't think this is a very big deal right now (although it would be useful to establish a pattern before the usage spreads too much). So just to give another point of view, to me in most cases the auxiliary functions seems to be very connected to the SoALayout (or really to Views derived from Layout). And I don't see those very different from OO-style data format class definition (data + member functions operating on that data).

From `PixelTrackUtilities.h`, `TrackingRecHitUtilities.h`, `ZVertexUtilities.h`, and `PixelVertexWorkSpaceUtilities` the only part that might be "too much" for a Layout-defining header is `pixelTrackSoA::QualityCutsT`.

But perhaps this question is something that would be good to discuss elsewhere to establish a pattern.